pkg/assets/create/creater.go

```diff
@@ -57,45 +55,50 @@ func EnsureManifestsCreated(ctx context.Context, manifestDir string, restConfig
 		options.StdErr = os.Stderr
 	}
 
-	// Default QPS in client (when not specified) is 5 requests/per second
-	// This specifies the interval between "create-all-resources", no need to make this configurable.
-	interval := 200 * time.Millisecond
-
-	// Retry creation until no errors are returned or the timeout is hit.
 	var (
 		lastCreateError      error
-		retryCount           int
 		mapper               meta.RESTMapper
 		needDiscoveryRefresh bool = true
 	)
-	err = wait.PollImmediateUntil(interval, func() (bool, error) {
-		retryCount++
+	for retryCount := 1; ; retryCount++ {
```

Comment:
`wait.PollImmediateUntil` is the canonical kubernetes way of such a loop. I would stay with it.

Reply:
Yeah, it is used and well tested in many kube components and packages.

Reply:
> `wait.PollImmediateUntil` is the canonical kubernetes way of such a loop. I would stay with it.

And just set the delay to zero?  It doesn't seem useful enough if all it does is wrap an initial `Context` check.